packages/core/src/BinaryDataManager/FileSystem.ts

```diff
@@ -11,7 +11,6 @@ import type { IBinaryDataConfig, IBinaryDataManager } from '../Interfaces';
 import { FileNotFoundError } from '../errors';
 
 const PREFIX_METAFILE = 'binarymeta';
```

Comment:
I'm wondering if we should also rename this to `flagged` or something like that to make this code less confusing.